packages/gestalt/src/Callout.js

```diff
@@ -12,15 +12,72 @@ import typography from './Typography.css';
 import { useColorScheme } from './contexts/ColorScheme.js';
 import styles from './Callout.css';
 import useResponsiveMinWidth from './useResponsiveMinWidth.js';
-import { type ActionDataType, type DismissButtonType, ActionDataPropType } from './commonTypes.js';
+import { type ActionDataType, ActionDataPropType } from './commonTypes.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
 
 type Props = {|
-  dismissButton?: DismissButtonType,
+  /**
+   * Adds a dismiss button to Callout. See the [Dismissible variant](https://gestalt.pinterest.systems/callout#Dismissible) for more info.
+   * The `accessibilityLabel` should follow the [Accessibility guidelines](https://gestalt.pinterest.systems/callout#Accessibility).
+   */
+  dismissButton?: {| accessibilityLabel: string, onDismiss: () => void |},
+  /**
+   * Label to describe the icon’s purpose. See the [Accessibility guidelines](https://gestalt.pinterest.systems/callout#Accessibility) for details on proper usage.
+   */
   iconAccessibilityLabel: string,
+  /**
+   * Main content of Callout. Content should be [localized](https://gestalt.pinterest.systems/callout#Localization).
+   *
+   * See [Best Practices](https://gestalt.pinterest.systems/callout#Best-practices) for more info.
+   */
   message: string,
-  primaryAction?: ActionDataType,
-  secondaryAction?: ActionDataType,
+  /**
+   * Main action for users to take on Upsell. If `href` is supplied, the action will serve as a link. See [OnLinkNavigationProvider](https://gestalt.pinterest.systems/callout/OnLinkNavigationProvider) to learn more about link navigation.
```

Comment:
Main action for users to take on Callout

Reply:
this is currently in the docs 😱 🤦‍♂️